api/src/main/java/module-info.java

```diff
@@ -14,16 +14,16 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  */
 
-module java.json.bind {
-    exports javax.json.bind;
-    exports javax.json.bind.adapter;
-    exports javax.json.bind.annotation;
-    exports javax.json.bind.config;
-    exports javax.json.bind.serializer;
-    exports javax.json.bind.spi;
+module jakarta.json.bind {
+    exports jakarta.json.bind;
+    exports jakarta.json.bind.adapter;
+    exports jakarta.json.bind.annotation;
+    exports jakarta.json.bind.config;
+    exports jakarta.json.bind.serializer;
+    exports jakarta.json.bind.spi;
 
     requires java.json;
```

Comment:
Is the name for the JSON-P module still `java.json`? Or did it change to `jakarta.json`? We should remain consistent with whatever JSON-P (and the rest of the Jakarta EE platform does)

Reply:
It should be changed to `jakarta`.

Reply:
Already done with jsonp 2.0.0-RC1.